com.google.Earth.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.google.Earth</id>
+  <name>Google Earth</name>
+  <developer_name>Google</developer_name>
+  <summary>Explore the far reaches of the world</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://www.google.com/earth/desktop/</url>
+  <description>
+    <p>Create maps with advanced tools on your PC.</p>
+    <ul>
+      <li>Compute distances and areas using measurement tools</li>
+      <li>Visualise, manipulate and export GIS data</li>
+      <li>Use Movie Maker to produce media collateral</li>
+      <li>Manipulate and export GIS data</li>
+      <li>Go back in time with historical imagery</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot>
+      <image type="source">https://www.google.com/earth/outreach/images/tutorials_earthoverlays5.jpg</image>
+    </screenshot>
+  </screenshots>
+  <!-- These are the icons for the android apps, from https://play.google.com/store/apps/details?id=com.google.earth-->
+  <icon type="remote" height="64" width="64">https://lh3.googleusercontent.com/AEUm_nsufnYaRVkZ4opiyGVWqbcbo7PF1fWxQvxGSaA5G4kN5s80oxf16pwkP6aOHV8=w64</icon>
+  <icon type="remote" height="128" width="128">https://lh3.googleusercontent.com/AEUm_nsufnYaRVkZ4opiyGVWqbcbo7PF1fWxQvxGSaA5G4kN5s80oxf16pwkP6aOHV8=w128</icon>
```

Comment:
Can be dropped (I guess this is a leftover).

Reply:
Correct!